include/d/a/obj/d_a_obj_shrine_after.h

```diff
@@ -6,7 +6,15 @@
 class dAcOshrineAfter_c : public dAcObjBase_c {
 public:
     dAcOshrineAfter_c() {}
+    virtual int create() override;
+    virtual int doDelete() override;
+    virtual int draw() override;
+    virtual bool createHeap() override;
     virtual ~dAcOshrineAfter_c() {}
+    virtual int actorExecute() override;
+
+    nw4r::g3d::ResFile mResFile;
+    m3d::smdl_c mModel;
```

Comment:
please add some offset comments at the front :)

Reply:
All sorted, went back and added them for the other stuff I did today as well :p